src/quacc/runners/prep.py

```diff
@@ -64,6 +65,8 @@ def calc_setup(
         symlink.symlink_to(tmpdir, target_is_directory=True)
 
     # Copy files to tmpdir and decompress them if needed
```

Comment:
With the change introduced in `copy_decompress_files` you can likely shorten this to

```python
if isinstance(copy_files, dict):
    copy_decompress_tree(copy_files, tmpdir)
if isinstance(copy_files, list):
    copy_decompress_files(copy_files, tmpdir)
elif isinstance(copy_files, (str, Path)):
    copy_decompress_files([copy_files], tmpdir)
```

if I'm understanding it correctly. No need to call `copy_decompress_files_from_dir` directly since that will be called from within `copy_decompress_files` if the list item `is_dir()`.